terraform/amazon/templates/modules.tf.template

```diff
@@ -115,6 +115,7 @@ module "vault" {
   vault_version = "${var.vault_version}"
   vault_root_size = "${var.vault_root_size}"
   vault_data_size = "${var.vault_data_size}"
+  vault_volume_encrypted = "${var.vault_volume_encrypted}"
```

Comment:
Use the encrypted directly in the templates, as almost all nodes/instances should be templated in our terraform code